helm/hpcc/templates/storage.yaml

```diff
@@ -43,4 +43,50 @@
  {{- end -}}
  {{- $_ := set $previousMounts $plane.prefix $rawPlane -}}
 {{- end -}}
+{{- /* check that planes exist for each of the requried planes */ -}}
+{{ $need := dict }}
+{{- range $.Values.dali -}}
+ {{- if not .disabled -}}
+   {{- $_ := set $need "dali" true -}}
+ {{- end }}
+{{ end -}}
+{{- range $.Values.eclccserver -}}
+ {{- if not .disabled -}}
+   {{- $_ := set $need "dll" true -}}
+ {{- end }}
+{{ end -}}
+{{- range $.Values.eclscheduler -}}
+ {{- if not .disabled -}}
+   {{- $_ := set $need "dali" true -}}
+   {{- $_ := set $need "dll" true -}}
+ {{- end }}
+{{ end -}}
+{{- range $.Values.esp -}}
+ {{- if not .disabled -}}
+ {{- end }}
+{{ end -}}
```

Comment:
looks like above 4 lines can/should be removed? or here as a sort of placeholder?

Reply:
yes, was here as a placeholder.  I will add a comment